src/main/java/htsjdk/samtools/cram/encoding/reader/CramRecordReader.java

```diff
@@ -34,7 +34,7 @@
 
 public class CramRecordReader {
     private final DataSeriesReader<Integer> bitFlagsCodec;
-    private final DataSeriesReader<Byte> compressionBitFlagsCodec;
+    private final DataSeriesReader<Integer> compressionBitFlagsCodec;
```

Comment:
Good catch on these. Did you by any chance compare the rest of them to the spec to see if there are any others ?

Reply:
Thx - the rest of the spec matches.  Though QS and BA are a little weird because they're sometimes byte and sometimes byte array.  (See 10.6 vs 10.7)